src/main/java/org/whispersystems/textsecuregcm/controllers/AccountController.java

```diff
@@ -339,6 +343,13 @@ private void createAccount(String number, String password, String userAgent, Acc
     if (accounts.create(account)) {
       newUserMeter.mark();
     }
+    if (cdsSender.isPresent()) {
+      try {
+        cdsSender.get().addRegisteredUser(number);
+      } catch (Throwable t) {
```

Comment:
Is there ever a case where we want this method to throw an exception to the caller? If not, maybe we can do this in the method itself?

Reply:
done in d719b604347590ffa7c9f8d85ee95d8046e24acf